### DISCUSSION TITLE
Need to return 200 on OPTIONS for AuthType  auth-openidc when session cookie not available

### DISCUSSION THREAD

**Discussion by abg35** [OpenIDC/mod_auth_openidc#647] [General]
When the AuthType is oauth20, the module returns a 200 when it receives an OPTIONS request, since OPTIONS requests do not send the Authorization header.  

However, when the AuthType is auth-openidc, an OPTIONS request is unauthorized unless there is an OIDC session cookie.  This means that the auth-openidc AuthType doesn't work for cors, since many browsers first send the OPTIONS request.  Requesting that for OPTIONS request with auth-openidc, it also returns a 200, or that it can be configured to do so.

**Comment by zandbelt** [comment]
sorry it took a while to get to this, but this is now addressed in https://github.com/zmartzone/mod_auth_openidc/commit/c675949ca3609c9fe7dc0bccdaa093021ac59cbe
